kanidmd/src/lib/utils.rs

```diff
@@ -39,11 +39,10 @@ pub fn password_from_random() -> String {
 }
 
 pub fn backup_code_from_random() -> HashSet<String> {
-    let mut set = std::collections::HashSet::with_capacity(8);
-    for _ in 1..8 {
-        set.insert(readable_password_from_random());
-    }
-    set
+    (1..9)
```

Comment:
Does this generate 8 codes? Just checking.

Reply:
It inserts 8 codes. But on second thought, if the `readable_password_from_random()` returns the same string, then the HashSet won't have 8 codes.

Reply:
I will change it to insert until set.size() <= 8

Reply:
Just make it 0..8, that's clearer .... Also readable_password_from_random returns a new random PW each call.